daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -2333,6 +2333,54 @@ def _resume_device(self):
         self.disable_notify = False
 
 
+class RazerBasiliskV3(__RazerDeviceSpecialBrightnessSuspend):
+    """
+    Class for the Razer Basilisk V3
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Basilisk_V3-if0(1|2)-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0099
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 11]
+    METHODS = ['get_device_type_mouse',
+               'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_dpi_stages', 'set_dpi_stages',
+               'get_poll_rate', 'set_poll_rate',
+               'get_brightness', 'set_brightness',
+               # Can set custom matrix effects
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://assets2.razerzone.com/pages/basilisk-v3-4D578898E8144Le8da21dde32b7a9f5f/basilisk.png"
+
+    DPI_MAX = 26000
+
+    def _suspend_device(self):
+        """
+        Suspend the device
+
+        Get the current brightness level, store it for later and then set the brightness to 0
+        """
+        self.suspend_args.clear()
+        self.suspend_args['matrix_brightness'] = (_get_backlight_brightness(self))
+
+        # Todo make it context?
+        self.disable_notify = True
+        _set_backlight_brightness(self, 0)
+        self.disable_notify = False
+
+    def _resume_device(self):
+        """
+        Resume the device
+
+        Get the last known brightness and then set the brightness
+        """
+        matrix_brightness = self.suspend_args.get('matrix_brightness', (100, 100, 100))[0]
```

Comment:
```suggestion
        brightness = self.suspend_args.get('brightness', 100)
```
and of course change the variable usage further down

Reply:
As stated above, a tuple is actually stored now, so the deconstruction makes more sense now.